### ISSUE TITLE
Visitors format change

### ISSUE THREAD

**Issue by offlinehacker** [kernc/www#4]
Use some other character as `#` for comment separation and also don't use `:` as number separator in the comment

**Comment by kernc** [comment]
Why? If prose strips `#` and `:` in text fields, does prose strip all special characters?

**Comment by offlinehacker** [comment]
Yes, it looks like. It does put them in quotation marks `"` and `"`

**Comment by kernc** [comment]
That's just standard YAML. Those quotes are required, but won't (shouldn't) be shown after the YAML is parsed (provided it's YAML and not simple RFC822 header metadata).

But inconsistencies might make for some pain when parsing with other tools. Can you make prose.io quote the whole string once?
